### PR TITLE
Add the possibility to visualize language instructions in visualize_dataset_html.py

### PR DESCRIPTION
**Pull request by michel-aractingi**
## What this does
Adds a line under the video visualization in `visualize_dataset_html.py` to print the language instuction if it exists in the dataset.

Example from the `nyu_franka_play_dataset`:
Screenshot 2024-08-28 at 08 58 20

## How it was tested
I tested it by running `python lerobot/scripts/visualize_dataset_html.py` with several datasets from OpenX  and other datasets that don't contain language instruction like pusht.

To test it yourself:
NYU Franka play dataset
```
python lerobot/scripts/visualize_dataset_html.py --repo-id lerobot/nyu_franka_play_dataset --episodes 0
```
CMU Stretch dataset
```
python lerobot/scripts/visualize_dataset_html.py --repo-id lerobot/cmu_stretch --episodes 0 1 2 3
```
Imperial College Sawyer dataset
```
python lerobot/scripts/visualize_dataset_html.py --repo-id lerobot/imperialcollege_sawyer_wrist_cam --episodes 0 1 2 3
```

For a sanity check, try it with a dataset that doesn't have language instruction pusht:
```
python lerobot/scripts/visualize_dataset_html.py --repo-id lerobot/pusht --episodes 0 1 2 3
```

### CODE REVIEW

File: lerobot/templates/visualize_dataset_template.html
Comment (Cadene) [inline lerobot/templates/visualize_dataset_template.html:108]: @mishig25 Does it look good to you? :)

Screenshot_2024-08-28_at_10 46 22

**Review by Cadene — APPROVED** [review]
LGTM, waiting for review from @mishig25 before merging

File: lerobot/templates/visualize_dataset_template.html
Comment (mishig25) [inline lerobot/templates/visualize_dataset_template.html:108]:
```suggestion
        {% if videos_info[0].language_instruction %}
        <div class="text-sm font-bold mt-2">
            Language Instruction: <em>{{ videos_info[0].language_instruction }}</em>
        </div>
        <hr class="my-4 border-gray-600">
```

Comment (mishig25) [inline lerobot/templates/visualize_dataset_template.html:108]:
```suggestion
        <p class="font-medium mt-2">
            Language Instruction: <span class="italic">{{ videos_info[0].language_instruction }}</span>
        </p>
```

looks good to me. if I'd suggest a stylistic change, it would be:
1. remove the border
2. change the text to be regular size and sembold rather than small text fully bold

Comment (mishig25) [reply lerobot/templates/visualize_dataset_template.html:108]: this is how the suggested change is lookign

image

**Review by mishig25 — APPROVED** [review]

**Pull request merged**
